### ISSUE TITLE
Library still supported?

### ISSUE THREAD

**Issue by MFlisar** [gabrielemariotti/changeloglib#43]
Pull requests are not merged so I'm asking this question...

I still like the lib and added following (or am adding this currently):
* support filtering logs (like show only logs that have not been shown yet)
* support custom log formatter in code

And I would make a pull request then...

**Comment by hannesa2** [comment]
Not merging is a disaster ! (my PR is also pending)
I like your ideas of your PR's, I want it too. But how to proceed ? 
The only way I see is to separate from the inactive maintainer and I see you did it already https://github.com/MFlisar/changelog

**Comment by MFlisar** [comment]
Correct. My library is a complete rewrite based on my own needs adjusted to what I personally consider useful or not
